packages/test.sh

```diff
@@ -17,6 +17,8 @@ virtualenv -p "$TEST_PYTHON" "$TEST_ENV_DIR"
 pip install --upgrade pip 'setuptools<58' wheel
 pip install -r../lib/galaxy/dependencies/pinned-lint-requirements.txt
 
+PIP_INDEX_OPTIONS="--index-url https://wheels.galaxyproject.org/simple --extra-index-url https://pypi.python.org/simple"
+
```

Comment:
I think we don't want this... the packages are supposed to be vanilla, pip installable things.

Reply:
I think (most?) our wheels don't have anything special, they are just pre-built, no? I was trying to save the time needed to build e.g. mercurial, since I don't think we have a pip cache on CircleCI.
But I can revert this and we can instead move `test_galaxy_packages` to GH workflows and use the pip cache there.

Reply:
OK, I will remove this later and see if it makes sense to move to GH workflows.

Reply:
Done in a rebase.